web/src/components/onboarding-modal/onboarding-modal.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import { useState } from 'react';
+import { Button } from '../ui/ui';
+import { PlayOutlineGreenIcon, MicIcon, SkipIcon } from '../ui/icons';
+import Modal from '../modal/modal';
+import URLS from '../../urls';
+import { LocaleLink } from '../locale-helpers';
+import './onboarding-modal.css';
+
+const STEPS: Array<any> = [
```

Comment:
Not a blocker, but it would be nice to add a type here.

```
interface StepTypeWithoutImage {
  title: string;
}
interface StepTypeWithImage extends StepTypeWithoutImage {
  img: string;
  alt: string;
}
type StepType = StepTypeWithoutImage | StepTypeWithImage;
```